supabase/tests/stats.test.sql

```diff
@@ -34,7 +29,7 @@ begin
       5, 1,
       0, 0
     );
-  insert into catalog_stat_partitions.bob_stats (
+  insert into catalog_stats (
```

Comment:
nit: insert multiple `values` tuples rather than re-doing the whole `insert into` stanza.